src/WinRT.Runtime/ExceptionHelpers.cs

```diff
@@ -374,18 +374,36 @@ public static unsafe void SetErrorInfo(Exception ex)
                         message = ex.GetType().FullName;
                     }
 
-                    IntPtr hstring;
+                    IntPtr hstringHeader = IntPtr.Zero;
+                    IntPtr hstring = IntPtr.Zero;
 
                     fixed (char* lpMessage = message)
                     {
-                        if (Platform.WindowsCreateString((ushort*)lpMessage, message.Length, &hstring) != 0)
+                        if (Platform.WindowsCreateStringReference(
+                            sourceString: (ushort*)lpMessage,
+                            length: message.Length,
+                            hstring_header: &hstringHeader,

```

Comment:
does the header need to be allocated?  See [here](https://github.com/microsoft/CsWinRT/blob/578f89272718974d140402f9d4e82d550ca082f5/src/WinRT.Runtime/Marshalers.cs#L134).

Reply:
Oh, oops yes you're right 😅